### ISSUE TITLE
3.8.5.5 Updating Our CI Pipeline / general pipeline contains specific branch instead of a main 

### ISSUE THREAD

**Issue by titoOdUA** [LukeMathWalker/zero-to-production#208]
Chapter: 3.8.5.5 Updating Our CI Pipeline
In this chapter we update our CI pipeline.

Problem:
After the changes our pipeline stops running.

Basically we have this text in a book
"You can find an updated version of the GitHub Actions setup on **GitHub**. Only general.yml needs to be
updated."

Github link redirects as to this page
https://github.com/LukeMathWalker/zero-to-production/blob/root-chapter-03-part1/.github/workflows/general.yml
Hooks in this version of pipeline configured on specific branch - root-chapter-03-part1 (i don't remember instructions to use this notation for the branch naming)

I suspect this is a mistake and needs to be fixed to use main branch (or maybe different pipeline altogether)?

So basically after the change our general pipeline never triggers.
I am not sure am I missing something or my version of the book is slightly wrong(bought about a month ago), or there indeed some problem

@LukeMathWalker

**Comment by titoOdUA** [comment]
Now that I think about it i guess I just use the book wrong and I suppose to just clone the repo and switch between branches instead of just coding along(maybe I have missed some instructions in this regard).

Maybe it is, indeed, false alarm, it just confused me =)

**Comment by LukeMathWalker** [comment]
No, you're supposed to code on your own!

The repository is there as a reference, but to accomodate multiple snapshots of the projects we are forced to use multiple branches, which in turn requires us to tweak the CI triggers slightly.

Ignore the trigger part (which is commented as such IIRC) and just look at the steps.
